app/controllers/events_controller.rb

```diff
@@ -0,0 +1,33 @@
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    current_user = User.find(session[:current_user_id])
+    @event = current_user.events.build(event_params)
+    # @event.date = 'Tomorrow'
```

Comment:
Please remove all commented out code 🙂 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂

Reply:
Thanks for the Feedback, @mcrd25,we have deleted the commented code as requested.